public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+  changes and at the same time may change the very same property's value.
+  
+  Instead of using a duplex communication channel between parent and child components, we can use two simplex channels, one from the parent to 
+  a child (or children), and another one in the opposite direction, from the child to its parent. While we can use input property binding
+  from parent to child, Angular offers event emitting (output event binding) for child-to-parent communication.
+  
+  ### Establishing output event binding
+  
+  To set up output event emitting, we need to carry out these steps:
+  1. Adding an outpt property to the child component (apply the `@Output()` decorator on the particular property)
+  1. Declaring this property as an `EventEmitter`
+  1. Using this property with the _output event binding syntax_ in the parent template to bind it to an event handler method
+  
+  The Hero Job Board application uses output event binding to sign that a hero undertakes the announced job. The `HeroPanel` component declares 
+  an output property as a type of `EventEmitter`, through which &mdash; as the type name suggests &mdash; it can emit an event. The parent 
+  `HeroJobBoard` component can use the output property to attach an event handler to that particular event.
+  
+  `HeroPanel` declares the `onJobTaken` output event property, as the `@Output` decorator indicates:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-panel.ts', 'component', 'take-job-event/hero-panel.ts (extract)', {pnk: /(@Output\(\)|takeJob\(\)|onJobTaken\.next)/g})
+
+:marked
+  The `takeJob()` method invokes the `next()` method of `onJobTaken`, and passes the `hero` assigned to this `HeroPanel`. 
+  The parameter of `next()` is the argument of the event emitted. Thanks to the strong type notation in TypeScript, `EventEmitter` is a generic type,
+  so we could declare `onJobTaken` as an `EventEmitter<Hero>`. When we invoke `next`, the TypeScript compiler checks whether the event argument 
+  is compatible with `Hero`.
+  
+  To use `@Output()` and `EventEmitter`, `HeroPanel` needs to import them:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-panel.ts', 'import', null, {blk: /(Output|EventEmitter)/g})
+
+:marked
+  Now our `HeroPanel` is ready to emit its event. `HeroJobBoard` attaches its `heroTakesJob()` event handler method to the `onJobTaken` output
+  event of `HeroPanel` using the output property binding syntax &mdash; wrapping the snake-case name of the output property, `on-job-taken`, into parentheses.
+  The output property binding expression passes the `$event` identifier &mdash; that represents the argument of the emitted event &mdash; to the handler method:
+  
++makeExample('component-communication/ts/app/take-job-event/hero-job-board.ts', 'component', 'take-job-event/hero-job-board.ts', {pnk: /(\(on-job-taken\))/g, blk: /(heroTakesJob\(hero: Hero\))/g})
```

Comment:
Too much code. Can't find the relevant part of the template.

I think we're going to have to move the HTML out of the component and into its own file so that we can use `<!-- #docregion -->` to cite the appropriate fragment.